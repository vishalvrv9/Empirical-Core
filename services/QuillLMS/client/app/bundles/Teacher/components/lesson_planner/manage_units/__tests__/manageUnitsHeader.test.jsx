import React from 'react';
import { shallow } from 'enzyme';

import ManageUnitsHeader from '../manageUnitsHeader';

describe('ManageUnitsHeader component', () => {
  const wrapper = shallow(<ManageUnitsHeader />)

  it('renders', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
