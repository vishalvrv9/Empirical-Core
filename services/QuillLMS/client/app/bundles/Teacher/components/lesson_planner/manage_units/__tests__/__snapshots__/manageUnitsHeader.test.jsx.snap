// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ManageUnitsHeader component renders 1`] = `
<div
  className="manage-units-header"
>
  <h1>
    My Activity Packs
  </h1>
  <div
    className="header-content"
  >
    <p>
      Below you can see all of the activities that you have assigned to your students. To assign more activities, click on the button to the right.
    </p>
    <a
      href="/teachers/classrooms/assign_activities"
    >
      <button
        className="button-green create-unit featured-button"
      >
        Assign A New Activity
      </button>
    </a>
  </div>
</div>
`;
